.github/workflows/style.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.6', '3.7', '3.8']
+        python-version: ['3.8']
```

Comment:
One could argue that you can remove the entire matrix in this scenario :slightly_smiling_face:

Reply:
One could agree :p

Reply:
Definitely :smile:

Reply:
:joy: